src/utils/helpers/browser/browser.js

```diff
@@ -89,6 +98,16 @@ const Browser = {
   },
 
   /**
+   * Sets the body scroll
+   *
+   * @param {Boolean} enabled
+   * @method setBodyScroll
+   */
+  setBodyScroll: (enabled) => {
+    Browser.getBody().style.overflow = enabled ? '' : 'hidden';
```

Comment:
[Minor] Do we think that this should be adding a class instead of setting a style on the object. Means we could possibly reuse it or add more properties.